src/app/content/components/Sidebar.tsx

```diff
@@ -160,21 +172,150 @@ const NavItemComponent: ComponentType<{active?: boolean, className?: string}> =
   >{children}</li>
 );
 
+// tslint:disable-next-line:variable-name
+const ContentLink = styled(ContentLinkComponent)`
+  ${textStyle}
+  display: flex;
+  margin-left: ${iconPadding + iconSize}rem;
+  text-decoration: none;
+
+  li[aria-label="Current Page"] & {
+    font-weight: bold;
+  }
+
+`;
+
 // tslint:disable-next-line:variable-name
 const NavItem = styled(NavItemComponent)`
+  ${textStyle}
   list-style: none;
   font-size: 1.4rem;
-  ${(props) => props.active && css`
-    overflow: visible;
-    position: relative;
+  line-height: 1.6rem;
+  overflow: visible;
+  margin: 0 0 1.5rem 0;
+
+  ${SummaryTitle}, ${ContentLink} {
+    :hover {
+      color: ${theme.textColors.black}
+    }
+  }
+
+`;
+
+const expandCollapseIconStyle = css`
+  height: ${iconSize}rem;
+  width: ${iconSize}rem;
+  margin-right: ${iconPadding}rem;
+`;
+
+// tslint:disable-next-line:variable-name
+const ExpandIcon = styled(CaretRight)`
+  ${expandCollapseIconStyle}
+`;
+
+// tslint:disable-next-line:variable-name
+const CollapseIcon = styled(CaretDown)`
+  ${expandCollapseIconStyle}
+`;
+
+// tslint:disable-next-line:variable-name
+const Summary = styled.summary`
+  display: flex;
+  list-style: none;
+  cursor: pointer;
+
+  &::-webkit-details-marker {
+    display: none;
+  }
+`;
+
+const getNumberWidth = (contents: ArchiveTree['contents']) => contents.reduce((result, {title}) => {
+  const num = splitTitleParts(title)[0];
+
+  if (!num) {
+    return result;
+  }
+
+  const numbers = num.replace(/[^0-9]/, '');
+  const periods = num.replace(/[^\.]/, '');
+
+  return Math.max(result, numbers.length * numberCharacterWidth + periods.length * numberPeriodWidth);
+}, 0);
+
+const getDividerWidth = (section: ArchiveTree) => {
 
-    :before {
-      font-weight: bold;
-      content: '>';
-      position: absolute;
-      right: 100%;
+  if (section.title.includes(' | ')) {
+    return 0.6;
+  }
+
+  return 0.4;
+
+};
+
+// tslint:disable-next-line:variable-name
+const NavOl = styled.ol<{section: ArchiveTree}>`
+  margin: 0;
+  padding: 0rem 3rem 0 0;
+
+  .os-number,
+  .os-divider,
+  .os-text {
+    display: inline-block;
+    overflow: hidden;
+  }
```

Comment:
these styles can go on some variable that contentLink and summaryTitle share?

Reply:
I think these styles don't really do anything, i'll remove them.